src/main/resources/assets/minestuck/models/block/retractable_spikes_extended.json

```diff
@@ -0,0 +1,616 @@
+{
+	"credit": "Made with Blockbench by Dweblenod",
+	"textures": {
+		"spikes": "minestuck:block/spikes",
+		"main_body": "minestuck:block/retractable_spikes_extended",
+		"particle": "minestuck:block/spikes"
+	},
+	"elements": [
+		{
+			"from": [0, 0, 0],
+			"to": [16, 16, 16],
+			"faces": {
+				"north": {"uv": [0, 0, 8, 16], "texture": "#main_body"},
+				"east": {"uv": [0, 0, 8, 16], "texture": "#main_body"},
+				"south": {"uv": [0, 0, 8, 16], "texture": "#main_body"},
+				"west": {"uv": [0, 0, 8, 16], "texture": "#main_body"},
+				"up": {"uv": [8, 0, 16, 16], "texture": "#main_body"},
+				"down": {"uv": [0, 0, 8, 15.5], "texture": "#main_body"}
+			}
+		},
+		{
+			"from": [1.00503, 22, 2],
+			"to": [3.00503, 23, 2],
```

Comment:
I'm assuming that these should be rounded to `[1, 22, 2]` and `[3, 23, 2]` (same goes for other values in this model with more than one number after the dot)